extensions/scarb-snforge-test-collector/src/metadata.rs

```diff
@@ -93,7 +93,14 @@ impl CompilationUnit<'_> {
                         } else {
                             Edition::default()
                         },
-                        experimental_features: Default::default(),
+                        experimental_features: self
+                            .metadata
+                            .get_package(&component.package)
+                            .unwrap_or_else(|| {
+                                panic!("Failed to find = {} package", component.package)
+                            })
```

Comment:
Since this pattern is used more than once - maybe helper? (e.g. `fetch_package`)

Reply:
Changed `get_package` to return `&PackageMetadata` with the `unwrap_or_else` call since the only other usage of this method was the same, but with a slightly differently phrased error in `panic!`.